improved documentation (#230)
Pepperi-Addons/papi-sdk: README.md

etc. See https://developer.pepperi.com/account-resources/apis/get/accounts

> When adding such and endpoint inherit one of the base classes `IterableEndpoint` and `Endpoint` to implement your API. 

Author: ```suggestion
When adding such an endpoint inherit one of the base classes
```
